AutomatedTesting/Assets/AutomatedTestScripts/frametime.sa.lua

```diff
@@ -0,0 +1,62 @@
+----------------------------------------------------------------------------------------------------
+--
+-- Copyright (c) Contributors to the Open 3D Engine Project.
+-- For complete copyright and license terms please see the LICENSE at the root of this distribution.
+--
+-- SPDX-License-Identifier: Apache-2.0 OR MIT
+--
+--
+--
+----------------------------------------------------------------------------------------------------
+-- optional settings
+local AssetLoadFrameIdleCountRegistryKey <const> = "/O3DE/ScriptAutomation/FrameTime/AssetFrameIdleCount"
+local FrameIdleCountRegistryKey <const> = "/O3DE/ScriptAutomation/FrameTime/IdleCount"
+local FrameCaptureCountRegistryKey <const> = "/O3DE/ScriptAutomation/FrameTime/CaptureCount"
+local ViewportWidthRegistryKey <const> = "/O3DE/ScriptAutomation/FrameTime/ViewportWidth"
+local ViewportHeightRegistryKey <const> = "/O3DE/ScriptAutomation/FrameTime/ViewportHeight"
```

Comment:
Was there supposed to be a new .setreg file to accompany these changes?

Reply:
My expectation is that these would be passed on the command line, or possibly in a script runtime generated setreg file.
On the command line each one would look like `--regset="/O3DE/ScriptAutomation/FrameTime/ProfileName=10KCubeFrameTime"`